alkira/resource_alkira_connector_gcp_vpc.go

```diff
@@ -75,10 +75,16 @@ func resourceAlkiraConnectorGcpVpc() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"id": {
-							Description: "The Id of the subnet.",
+							Description: "The numeric Id of the subnet. This is the Id returned by the GCP REST API.",
```

Comment:
Could we tweak all `Id` to `ID` to be consistent across?

Reply:
Done